app/src/main/res/layout/project_search_result_view.xml

```diff
@@ -24,8 +26,7 @@
       android:id="@+id/project_image_view"
       android:scaleType="centerCrop"
       android:background="@drawable/gray_gradient"
-      android:layout_alignParentEnd="true"
-      android:layout_alignParentRight="true"
+      android:layout_alignParentStart="true"
```

Comment:
should also have an accompanying `android: layout_alignParentLeft` for < 21 apis